CRM/Report/Form/Grant/Statistics.php

```diff
@@ -487,7 +487,7 @@ public function alterDisplay(&$rows) {
                 "({$values['percentage']}%)";
             }
             $totalAmt = implode(', ', $totalAmount);
-            $count = (boolean) CRM_Utils_Array::value('count', $values, 0) ? $values['count'] . " ({$values['percentage']}%)" : '';
+            $count = (bool) CRM_Utils_Array::value('count', $values, 0) ? $values['count'] . " ({$values['percentage']}%)" : '';
```

Comment:
Wow this line gets a :trophy: for unnecessary complexity. How about:
```suggestion
            $count = !empty($values['count']) ? $values['count'] . " ({$values['percentage']}%)" : '';
```

Reply:
Right. I read it another 3 times and then rearranged it slightly to reverse the polarity (which always works) and remove the string concatenation.